phpBB/phpbb/install/helper/iohandler/cli_iohandler.php

```diff
@@ -125,6 +125,10 @@ public function add_error_message($error_title, $error_description = false)
 	{
 		$this->io->newLine();
 
+		if (strpos($error_title, '<br />') !== false)
+		{
+			$error_title = strip_tags(str_replace('<br />', "\n", $error_title));
```

Comment:
SQL errors will be shown using HTML. That way the <br /> tags will be converted to new lines and then all other html tags will be removed.